### PR TITLE
Add build invocation ID

### PR DESCRIPTION
**Pull request by ianlewis**
Updates #8

### CODE REVIEW

File: slsa/provenance.go
Comment (asraa) [inline slsa/provenance.go:68]: i'm just curious about this, but is a workflowrun materials really completeness true (from `NewWorkflowRun`)?

For generic generator, I'm not sure

File: slsa/provenance.go
Comment (asraa) [inline slsa/provenance.go:48]: Could you please add some documentation on the format of this field? 

(Maybe add here https://github.com/slsa-framework/slsa-github-generator#provenance-example and add a note)

**Review by asraa — COMMENTED** [review]
Thanks!

File: slsa/provenance.go
Comment (ianlewis) [reply slsa/provenance.go:68]: Completeness gets taken from the `WorkflowRun`. The generic generator only supplies `completeness.parameters = true` based on a suggestion from @MarkLodato. All the rest of the fields are false.

https://github.com/slsa-framework/slsa-github-generator/blob/main/slsa/github.go#L117-L119

See https://github.com/slsa-framework/slsa-github-generator/pull/3#discussion_r840747519 for context when it was added.

File: slsa/provenance.go
Comment (ianlewis) [reply slsa/provenance.go:48]: I will add something related to the invocation ID. I also added an issue to better document the fields determined by the `buildType` where we have made some judgements about the data there. 
https://github.com/slsa-framework/slsa-github-generator/issues/33

Comment (ianlewis) [reply slsa/provenance.go:48]: Done.

File: slsa/provenance.go
Comment (ianlewis) [reply slsa/provenance.go:68]: I'm going to resolve as I think there isn't anything to do here but feel free to reopen if necessary.

**Review by laurentsimon — APPROVED** [review]

**Comment by laurentsimon** [comment]
Ready to merge?

**Comment by ianlewis** [comment]
> Ready to merge?

Yep. Just waiting for an approval.

**Pull request merged**
